src/systems/ec/implementations/vanilla/packages/ec-component-gallery/ec-component-gallery.js

```diff
@@ -423,6 +439,38 @@ export class Gallery {
       }
     }
 
+    // Update counter
+    this.overlayCounterCurrent.innerHTML =
+      +selectedItem.getAttribute('data-ecl-gallery-item-id') + 1;
+    this.overlayCounterMax.innerHTML = this.galleryItems.length;
+
+    // Update share link
+    const shareHref = this.selectedItem.getAttribute(
+      'data-ecl-gallery-item-share'
+    );
+    if (shareHref != null) {
+      this.overlayShare.href = shareHref;
+      this.overlayShare.style.display = 'inline-block';
```

Comment:
I'm not a big fan of inline style handled by js. I'd recommand using a css class or some html attributes like `hidden`